fhir-bucket/src/main/java/com/ibm/fhir/bucket/cos/CosClient.java

```diff
@@ -41,6 +46,9 @@
  */
 public class CosClient {
     private static final Logger logger = Logger.getLogger(CosClient.class.getName());
+
```

Comment:
COS should also be capitalized per convention.

Reply:
ok